nautobot/ipam/tests/test_api.py

```diff
@@ -222,7 +222,7 @@ def test_create_single_available_prefix(self):
             response = self.client.post(url, data, format="json", **self.header)
             self.assertHttpStatus(response, status.HTTP_201_CREATED)
             self.assertEqual(response.data["prefix"], str(prefixes_to_be_created[i]))
-            self.assertEqual(str(response.data["namespace"]), self.absolute_api_url(prefix.namespace))
+            self.assertEqual(str(response.data["namespace"]["url"]), self.absolute_api_url(prefix.namespace))
```

Comment:
optional, but shall we check the `object_type` and the `id`  attributes are rendered correctly as well?
Edit: This response is `depth=0` correct? If it is `depth=1`, then nvm.

Reply:
The base test case `test_list_objects_depth_0` should be covering most of this. This is just validating the data that was returned. I feel that's redundant.